### PR TITLE
Airspace Modification in VATPRC airspace

### PR DESCRIPTION
**Pull request by websterzh**

### CODE REVIEW

**Comment by Adrian2k** [comment]
Do you have any description of what changes you have made?

**Comment by websterzh** [comment]
> Do you have any description of what changes you have made?

They are simply just some accumulated adjustments to various FIR boundaries in China according to the latest AIRAC, also with some subsectors specified in the file.

**Review by NelisV — APPROVED** [review]
@websterzh As a general point, i think we should be careful of not adding to many sectors. As we've stated towards some other divisions: sectors have to be actively used and can't be event only sectors. In this case I've checked the numbers and do see these being used for quite a bit of hours, so as far as i'm concerned we can approve this. I would like to see some consolidation to sectors you are not using to prevent having to much clutter in the area if this is possible. It's already quite busy on the eyes. ZSJN/ZHCC/ZBHH?

**Review by Adrian2k — APPROVED** [review]

**Comment by websterzh** [comment]
> @websterzh As a general point, i think we should be careful of not adding to many sectors. As we've stated towards some other divisions: sectors have to be actively used and can't be event only sectors. In this case I've checked the numbers and do see these being used for quite a bit of hours, so as far as i'm concerned we can approve this. I would like to see some consolidation to sectors you are not using to prevent having to much clutter in the area if this is possible. It's already quite busy on the eyes. ZSJN/ZHCC/ZBHH?

**tl;dr** We are very likely not going to add more sectors into the dataset, except for boundaries adjustment or airport supplement.

I would definitely agree the idea of preventing the dataset to be overcomplicated, but I feel we do need to present accurate map on VATSpy to ensure most of our pilots will not feel confused when they receiving "contactme" message. Having said this, also to explain our consideration on this, when we consider the sectors that will appears on VATSpy we are not going to include those sub-sectors of ACCs such as ZSJN_1_CTR or ZHCC_2_CTR, except for those sub-sectors that have been delegated to different ACCs. China does have reasonable amount of ACCs considering how large the airspace is. For those ACC-specific positions like ZSJN_CTR and ZHCC_CTR , they provide more options to our controllers and are now possible to be used not just for the events. But for the consideration of service coverage, we will not further dividing them and hence there will not be more sub-sectors being added in near future.
